Tools/clang-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Perhaps OS/Platform specific tools should be in Firmware/Tools/<platform>/ to make it clear they are not intended for use on other platforms.